juju/delta.py

```diff
@@ -108,5 +108,7 @@ def get_entity_class(self):
     'relation': RelationDelta,
     'remoteApplication': RemoteApplicationDelta,
     'charm': CharmDelta,
-    'applicationOffer': ApplicationOfferDelta,
+    'applicationOffer': ApplicationOfferDelta
+    # TODO (stickupkid): Currently ModelDelta is missing from the all watcher,
```

Comment:
Let's create an issue for this to reference please. I'm a bit -1 on TODO without a bug/issue to go with it in general.

Reply:
https://github.com/juju/python-libjuju/issues/380